### ISSUE TITLE
Uncaught TypeError: Arguments to path.join must be strings

### ISSUE THREAD

**Issue by kdrachtidis** [AtomLinter/Linter#333]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.170.0
**System**: Mac OS X 10.10.1
**Thrown From**: [linter](https://github.com/AtomLinter/Linter) package, v0.7.3

### Stack Trace

Uncaught TypeError: Arguments to path.join must be strings

```
At path.js:390

TypeError: Arguments to path.join must be strings
  at Object.exports.join (path.js:390:15)
  at LinterJshint.Linter.getNodeExecutablePath (/Users/d059136/.atom/packages/linter/lib/linter.coffee:99:10)
  at LinterJshint.Linter.getCmdAndArgs (/Users/d059136/.atom/packages/linter/lib/linter.coffee:80:25)
  at LinterJshint.Linter.lintFile (/Users/d059136/.atom/packages/linter/lib/linter.coffee:107:24)
  at /Users/d059136/.atom/packages/linter/lib/linter-view.coffee:141:18
  at Object.oncomplete (fs.js:93:15)

```

### Commands

```
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v0.22.3
linter, v0.7.3
linter-csslint, v0.0.11
linter-jscs, v1.4.9
linter-jshint, v0.1.0
linter-php, v0.0.11

# Dev
No dev packages
```

/cc @atom/core

**Comment by smlombardi** [comment]
+1

**Comment by Lionardo** [comment]
+1

**Comment by dmnd** [comment]
Dupe of #329. This is fixed in v0.10.0, please upgrade.
